app_dart/bin/validate_task_ownership.dart

```diff
@@ -15,13 +15,19 @@ import 'package:http/http.dart' as http;
 ///
 /// This currently supports `flutter/flutter` only.
 Future<List<String>> remoteCheck(String repo, String ref) async {
-  final String ciYamlContent = await githubFileContent(
+  const String _kCocoonUseInMemoryCache = 'COCOON_USE_IN_MEMORY_CACHE';
+  final bool inMemoryCache = Platform.environment[_kCocoonUseInMemoryCache] == 'true';
+  final CacheService cache = CacheService(inMemory: inMemoryCache);
+
+  final Config config = Config(dbService, cache);
+  final GithubService githubService = await config.createDefaultGitHubService();
```

Comment:
For the bin and integration tests, you can avoid bringing in Config. Using `GithubService(Github())` is fine